projects/Qualcomm/devices/Dragonboard/README.md

```diff
@@ -0,0 +1,45 @@
+## Install Instructions
+
```

Comment:
could you add something like https://github.com/LibreELEC/LibreELEC.tv/blob/master/projects/Rockchip/devices/RK3328/README.md that we know how to build it.

Reply:
It's no different then any other platform to build. Only the install is different and that's why I documented it.